docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -1,29 +1,37 @@
-[[modules-remote-clusters]]
+[[remote-clusters]]
 == Remote clusters
-
 You can connect a local cluster to other {es} clusters, known as _remote
-clusters_. Once connected, you can search remote clusters using
-<<modules-cross-cluster-search,{ccs}>>. You can also sync data between clusters
-using <<xpack-ccr,{ccr}>>.
-
-To register a remote cluster, connect the local cluster to nodes in the
-remote cluster using one of two connection modes:
-
-* <<sniff-mode,Sniff mode>>
-* <<proxy-mode,Proxy mode>>
-
-Your local cluster uses the <<modules-network,transport layer>> to establish
-communication with remote clusters. The coordinating nodes in the local cluster
-establish <<long-lived-connections,long-lived>> TCP connections with specific
-nodes in the remote cluster. {es} requires these connections to remain open,
-even if the connections are idle for an extended period.
-
-You can use the <<cluster-remote-info, remote cluster info API>> to get
-information about registered remote clusters.
+clusters_. Remote clusters can be located in different datacenters or 
+geographic regions, and contain indices or data streams that can be replicated
+or searched by a local cluster. You can use remote clusters in {ccr} and {ccs}.
+
+With <<xpack-ccr,{ccr}>>, you index data to a leader index, which is 
+replicated to one or more read-only follower indices on a remote cluster.
+Creating a multi-cluster architecture with {ccr} enables you to configure
+disaster recovery, bring data closer to your users, or establish a centralized 
+reporting cluster to process reports locally.
+
+<<modules-cross-cluster-search,{ccs-cap}>> enables you to run a search request 
+against one or more remote clusters. This capability provides each region
+with a global view of all clusters, allowing you to send a search request from
+a local cluster and return results from all connected remote clusters. 
+
+Connected remote clusters effectively form a single security domain, where users 
+with administrative privileges in one cluster have those privileges in _all_ 
+connected clusters. This behavior means that you can't restrict a user's access 
+to a specific remote cluster. To use {ccr} or {ccs} safely with remote clusters, 
+<<remote-clusters-security,enable security>> on all connected clusters and
```

Comment:
I tried to distill this information into a few sentences -- please let me know if it's too verbose:
>When connecting a local cluster to a remote clusters, an {es} superuser (such as the `elastic` user) gains total read access to the remote clusters. Furthermore, a local user with administrative privileges at the 
operating system level can alter `elasticsearch.yml`. A malicious user with these privileges has potential access to private keys that could give them the ability to take over a remote cluster. To use {ccr} and {ccs} safely with remote 
clusters, <<remote-clusters-security,enable security>> on all connected clusters  and configure Transport Layer Security (TLS) on at least the transport level on every node.

Reply:
I'd like to make a few changes for the first 3 sentences as follows:

> Connected remote clusters effectively form a single security domain. A local {es} superuser (such as the elastic user) gains total read access to the remote clusters. Furthermore, a operating system level adminstrator can potentially take over a remote cluster if it has access to private keys and config files of the local cluster. 

Before advising enabling security in the last sentence, I think we should wrap up the above by emphasizing that we should treat local and remote cluster with equal importance when it comes to security (because they are effectively a single security domain). I'll leave it to you to paraphrasing. Thanks.

Reply:
I've modified this section as follows:

> Enabling and configuring security is important on both local and remote clusters. When connecting a local cluster to remote clusters, an {es} superuser (such as the `elastic` user) gains total read access to the remote clusters. Furthermore, a local user with administrative privileges at the operating system level can alter `elasticsearch.yml`. A malicious user with these privileges has potential access to private keys that could give them the ability to take over a remote cluster. To use CCR and CCS safely, enable security on all connected clusters and configure Transport Layer Security (TLS) on at least the transport level on every node.